### PR TITLE
BarChart and DataView components

### PR DESCRIPTION
**Pull request by heyitschun**

### CODE REVIEW

**Review by binismail — COMMENTED** [review]
looks good

File: zappnode-front/src/routes/Dashboard.js [hunk 1]
```diff
@@ -1,179 +1,21 @@
...
-      </Dialog>
+    <>
+      <div>
+        <DataView />
+      </div>
       <div className="mt-12">
         <DataTable />
       </div>
-    </div>
+      <div className="mt-12">
```
Comment (insuusvenerati) [inline zappnode-front/src/routes/Dashboard.js]: Let's add some padding to these so they aren't sitting on the edges. 

```suggestion
      <div className="mt-12 p-5">
```

File: zappnode-front/src/routes/Dashboard.js [hunk 2]
```diff
@@ -1,179 +1,21 @@
...
-      />
-
-      <Dialog header="Trackers" visible={visible} modal={true} onHide={() => setVisible(false)}>
-        {renderTrackerDialogContent()}
-      </Dialog>
+    <>
+      <div>
+        <DataView />
+      </div>
       <div className="mt-12">
```
Comment (insuusvenerati) [inline zappnode-front/src/routes/Dashboard.js]:
```suggestion
      <div className="mt-12 p-5">
```

File: zappnode-front/src/routes/Dashboard.js [hunk 3]
```diff
@@ -1,179 +1,21 @@
...
-        rows={3} // Value must be lower than `trackers.length`
-        sortOrder={sortOrder}
-        sortField={sortField}
-      />
-
-      <Dialog header="Trackers" visible={visible} modal={true} onHide={() => setVisible(false)}>
-        {renderTrackerDialogContent()}
-      </Dialog>
+    <>
+      <div>
```
Comment (insuusvenerati) [inline zappnode-front/src/routes/Dashboard.js]:
```suggestion
      <div className="mt-6 p-5">
```

**Review by insuusvenerati — CHANGES_REQUESTED** [review]
This looks very good. Just needs some padding :)

**Review by insuusvenerati — APPROVED** [review]

**Review by insuusvenerati — APPROVED** [review]

**Review by insuusvenerati — APPROVED** [review]

**Review by binismail — APPROVED** [review]
looks nice!! kudos

**Comment by insuusvenerati** [comment]
Trying to approve this but Github is being uncooperative. Sorry for the delay

**Comment by binismail** [comment]
> Trying to approve this but Github is being uncooperative. Sorry for the delay

you know, unless you are assigned you can't approve

**Review by aorelhas — APPROVED** [review]
Since the people from the frontend give good feedback, I'll merge.

**Pull request merged**